### ISSUE TITLE
Checkbox missing visual focus indicator using Tailwind theme

### ISSUE THREAD

**Issue by meixger** [formkit/formkit#1045]
### Reproduction

https://formkit.link/f48f9e10100e2ef6d2eb91b384db2dcb

### Describe the bug

Using the Tailwind theme no visual focus indicator is visible on the checkbox input.
Works find using the Genesis theme.

![image](https://github.com/formkit/formkit/assets/600424/8d5e9c15-1fc8-407b-909f-6c6efeb0dfc4)

### Environment

Version: 1.4.x (next), Tailwind
Browser: Chrome, Firefox

**Comment by meixger** [comment]
Resolved with the _regenesis_ theme from https://themes.formkit.com/

**Comment by justin-schroeder** [comment]
Agreed 👍❤️
